.evergreen/setup-oidc-roles.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+set -o errexit  # Exit the script with error if any of the commands fail
+set -o xtrace   # Write all commands first to stderr
+
+cd ${DRIVERS_TOOLS}/.evergreen/auth_oidc
+. ./activate_venv.sh
+
+# Install mongosh
+curl -L https://github.com/mongodb-js/mongosh/releases/download/v1.6.2/mongodb-mongosh-shared-openssl3-1.6.2.x86_64.rpm -o mongosh.rpm
+
+ls -la
+
+sudo yum -y install mongosh.rpm
```

Comment:
The script provided with drivers-evergreen-tools does not work with `mongosh`, only `mongo`. I felt it just better that we use the tools we currently support here instead of using the legacy 5.0 `mongo` that mongo orchestration still relies on.

Reply:
Should there be some sort of TODO (with a linked ticket if it exists)?

Reply:
Yes it does - I'll update.

Reply:
Updated.